.github/workflows/release.yaml

```diff
@@ -123,7 +120,9 @@ jobs:
 
       - name: Tag release
         run: |
-          git tag ${{ github.event.inputs.version }}
+          git config --global user.name "anchoreci"
```

Comment:
Why pass `--global`? Is this setting correct for every job that runs on this CI worker? (Maybe it is, come to think of it.)

Reply:
I can leave off global and it would also still function (I added it out of habit), but leaving the global flag will not affect other jobs as the github hosted CI runners don't allow for persisted state (you get a clean state every time).